index.js

```diff
@@ -3,22 +3,37 @@
 const fp = require('fastify-plugin')
 const elasticsearch = require('elasticsearch')
 
-function createClient (options) {
```

Comment:
I see no need to wrap it in a function we still have `fastifyElasticSearch` as stack trace.